### ISSUE TITLE
lose keyboard focus when using onboard, unable to open onboard setting

### ISSUE THREAD

**Issue by samchiu90** [localdesktop/localdesktop.github.io#315]
Cannot type with onboard: 
open terminal -> open onboard -> tap terminal for input -> tap onboard
then the input focus on the terminal lost so tapping onboard does not have any effect

Unable to open onboard setting:
when opening onboard setting from task bar, nothing happen.
After a while, it crash.

Device: Sony Xperia XZ Premium (G8141)
Android: 12 (LineageOS 19)
